python_bot.py

```diff
@@ -521,7 +521,8 @@ def emojify(string):
     foos_time = lunch_time + datetime.timedelta(seconds=30*60)
     mess_with_kevin = False
     thirtyMinWarning = False
-    await client.send_message(discord.Object(id=id_general), "Started with version: " + version)
```

Comment:
Are you wanting to remove this?

Reply:
Yea. instead of having the bot say in a specific channel id it's version now it just has it in it's `playing` status. Makes the bot more general

Reply:
oh I think i got u. +1